enterprise/server/githubapp/githubapp.go

```diff
@@ -146,6 +180,93 @@ func New(env environment.Env) (*GitHubApp, error) {
 	return app, nil
 }
 
+// GetGitHubAppForGroup returns the BB GitHub app that the current group has installed.
+// For now, a group can only have one app installed at a time (either read-write
```

Comment:
is there anything that enforces this?

Reply:
I was thinking that in our UI, we would only show buttons related to your current app. Theoretically if you initiated the install from github you could have two though

We could be more explicit about this to be more careful and have the FE always pass the app ID, but I felt like it wasn't necessary in practice. Don't feel strongly either way though

Reply:
I actually just added a check to prevent people from installing multiple github apps